### ISSUE TITLE
Provide third parties that interact with Notify

### ISSUE THREAD

**Issue by jimleroyer** [cds-snc/notification-planning#157]
### Task Description

List all third party vendors used by Notify under an agreed upon format with the security team.

This needs to occur as we are in the process of an ATO process. This is required make a security assessment on all tools.

### Task Outcome

Unblock ATO process by providing list of vendors used by Notify.

### Acceptance Criteria

- [ ] Agree with security team on a format for listing third party vendors.
- [ ] Provide the list to security team.
- [ ] Link it into the [risk register](https://docs.google.com/spreadsheets/d/1GSBptkZe1r1GoTNntGesI31B8eZkTQ4ItTdd172eeOw/edit#gid=0).

**Comment by jimleroyer** [comment]
Closing this as it appears a duplicate of #158.
